use-timescale/upgrades.md

```diff
@@ -12,186 +12,216 @@ import SupportMatrix from "versionContent/_partials/_migrate_self_postgres_times
-automatically by $CLOUD_LONG. To upgrade $TIMESCALE_DB manually, run `ALTER
-EXTENSION timescaledb UPDATE` in your $SERVICE_LONG, or `Pause` and `Resume`.
-If no action is taken by the user, the upgrade is triggered in the next
-available maintenance window. You can configure the maintenance window so that
-these upgrades are started at a particular time, on a set day of the week. 
+During a maintenance event, services with replicas perform maintenance on each node independently. When maintenance is 
+complete on the primary node, it is restarted: 
+- If the restart takes more than a minute, the replica node is promoted to primary, given that the replica has no 
+  replication lag. Maintenance now proceeds on the newly promoted replica, following the same
+  sequence. If the newly promoted replica takes more than a minute to restart, the former
+  primary is promoted back. In total, the process may result in up to two minutes of write
+  downtime and two failover events. 
+- If the maintenance on the primary node is completed within a minute and it comes back online, the replica remains
+  the replica.
 
-If there are no pending upgrades available during a regular maintenance window,
-no changes are performed.
 
-When you are considering your maintenance window schedule, you might prefer to
-choose a day and time that usually has very low activity, such as during the
-early hours of the morning, or over the weekend. This can help minimize the
-impact of a short service interruption. Alternatively, you might prefer to have
-your maintenance window occur during office hours, so that you can monitor your
-system during the upgrade.
+### Manually upgrade $TIMESCALE_DB for non-critical upgrades
 
-<Procedure>
+Non-critical upgrades are available before the upgrade is performed automatically by $CLOUD_LONG. To upgrade 
+$TIMESCALE_DB manually: 
 
-### Adjusting your maintenance window
+<Procedure>
 
-1.  [Log in to your Timescale account][cloud-login]. Click the name of the
-    service that you want to manage the maintenance window for.
-2.  In the `Operations` tab, navigate to the `Environment` > `Maintenance` and click `Change maintenance window`.
-3.  Select the day of the week, the time, and the
-    timezone that you want the maintenance window to start. Maintenance windows
-    can run for up to four hours.
+1. **Connect to your service**
 
-    ![Maintenance and upgrades](https://assets.timescale.com/docs/images/timescale-maintenance-upgrades.png)
+   In [$CONSOLE][cloud-login], select the $SERVICE_SHORT you want to upgrade.
 
+1. **Upgrade $TIMESCALE_DB**
 
-4.  Check `Apply new maintenance window to all services` if you want to use the
-    same maintenance window settings for all of your Timescale services.
-5.  Click `Apply`.
+   Either:
+   - Click `SQL Editor`, then run `ALTEREXTENSION timescaledb UPDATE`. 
+   - Click the kebab menu, then `Pause` and `Resume` the service.
```

Comment:
I don't think we use the name kebab menu anywhere, so I assume this should be the actual icon from the UI

Reply:
Found the icon, how cool.